### PR TITLE
feat: 🎸 settings page

### PR DESCRIPTION
**Pull request by DhariniJeeva**
✅ Closes: https://hashicorp.atlassian.net/browse/ICU-14398

# Description
This PR adds a skeleton for the settings page, note this does not include any functionality. 

## Screenshots (if appropriate)

Screenshot 2024-07-16 at 5 36 45 PM

## Checklist

- [x] I have added before and after screenshots for UI changes
- ~[ ] I have added JSON response output for API changes~
- ~[ ] I have added steps to reproduce and test for bug fixes in the description~
- ~[ ] I have commented on my code, particularly in hard-to-understand areas~
- ~[x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: ui/desktop/tests/acceptance/projects/settings/index-test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module } from 'qunit';
+
+module('Acceptance | projects | settings | index', function () {});
```
Comment (ZedLi) [inline ui/desktop/tests/acceptance/projects/settings/index-test.js:8]: Maybe we should have the first test be you can navigate to it

**Review by ZedLi — APPROVED** [review]
We should merge this to a llb

File: ui/desktop/app/templates/scopes/scope/projects.hbs [hunk 2]
```diff
@@ -15,6 +15,10 @@
         <Rose::Icon @name='flight-icons/svg/entry-point-16' @size='medium' />
         {{t 'resources.session.title_plural'}}
       </nav.link>
+      <nav.link @route='scopes.scope.projects.settings'>
+        <Rose::Icon @name='flight-icons/svg/settings-16' @size='medium' />
+        {{t 'titles.settings'}}
+      </nav.link>
```
Comment (cameronperera) [inline ui/desktop/app/templates/scopes/scope/projects.hbs:21]: Sorry if stated before but do we need a feature flag for this in-progress work?

Comment (DhariniJeeva) [reply ui/desktop/app/templates/scopes/scope/projects.hbs:21]: since we are merging this to a LLB I don't think we need one

Comment (cameronperera) [reply ui/desktop/app/templates/scopes/scope/projects.hbs:21]: Gotcha, thanks. I saw main as the base branch. Good with this as is going into an llb.

**Review by cameronperera — APPROVED** [review]

File: ui/desktop/tests/acceptance/projects/settings/index-test.js (hunk 1 above)
Comment (DhariniJeeva) [reply ui/desktop/tests/acceptance/projects/settings/index-test.js]: added!

**Review by calcaide — APPROVED** [review]

**Pull request merged**